3d_registration/learn2reg_oasis_unpaired_brain_mr.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !python -c "import monai" || pip install -q "monai-weekly[nibabel, tqdm]"
</pre><p>looks like can remove the unused nibabel and tqdm.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1566/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>